frontend/components/SetupPage/Create/SetupPassword.tsx

```diff
@@ -28,6 +30,7 @@ export const SetupPassword = () => {
       .then(({ mnemonic }: { mnemonic: string[] }) => {
         setMnemonic(mnemonic);
         goto(SetupScreen.SetupSeedPhrase);
+        setUserLoggedIn();
```

Comment:
we still have it in `SetupCreateSafe` so maybe now it makes sense to remove from there